Failed test on Windows due to obsolite gulp-jasmine dependency

Due to the hacky line
```js
// NOTE: this is really janky but a main function will be added in the next 2.x.x release
var jasmineRequire = require('../node_modules/jasmine-core/lib/jasmine-core/jasmine.js');
```
in minijasminenode2, `npm test` fails on Windows.

OS: Windows10
node: v5.7.1
npm: 3.6.0

Error:
```
> karma-jspm@2.0.3 test C:\Users\dotnet\Projects\opensource\karma-jspm
> gulp test

[12:54:43] Using gulpfile ~\Projects\opensource\karma-jspm\gulpfile.js
[12:54:43] Starting 'test'...
[12:54:43] 'test' errored after 27 ms
[12:54:43] Error: Cannot find module '../node_modules/jasmine-core/lib/jasmine-core/jasmine.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (C:\Users\dotnet\Projects\opensource\karma-jspm\node_modules\minijasminenode2\lib\index.js:5:22)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

--- a/package.json
+++ b/package.json
@@ -31,6 +31,6 @@
   },
   "devDependencies": {
     "gulp": "^3.8.7",
-    "gulp-jasmine": "^1.0.0"
+    "gulp-jasmine": "^2.3.0"
   }
 }

